package.json

```diff
@@ -2,21 +2,19 @@
   "name": "react-flip-move",
   "version": "2.10.0",
   "description": "Effortless animation between DOM changes (eg. list reordering) using the FLIP technique.",
-  "main": "lib/index.js",
+  "main": "dist/react-flip-move.cjs.js",
+  "module": "dist/react-flip-move.es.js",
   "typings": "typings/react-flip-move.d.ts",
+  "sideEffects": false,
   "files": [
     "dist",
-    "lib",
     "src",
     "typings"
   ],
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "build:lib": "babel src --out-dir lib",
-    "build:umd": "webpack src/index.js dist/react-flip-move.js --config webpack.config.production.js",
-    "build:umd:min": "webpack src/index.js dist/react-flip-move.min.js --config webpack.config.production.min.js",
-    "build": "npm run build:lib && npm run build:umd && npm run build:umd:min",
+    "build": "rollup -c",
```

Comment:
Curious why you changed `build:lib`? Seems like you'd want to leave the files unbundled for the NPM-distributed one, and only change the UMD build.

If I import this library in a webpack-built application, webpack will be bundling this library regardless; is the idea that because Rollup concatenates it into a single file, it would still be smaller in the host application, because it only has 1 webpack-module-wrapper chunk of code?

I also kinda like the idea of keeping the `lib` files as the ones being used in dev, so that you can poke around with the source in `node_modules` and not have to deal with one big file... although I suppose that's a pretty niche concern.

Reply:
i do it all the time 👼

Reply:
> If I import this library in a webpack-built application, webpack will be bundling this library regardless; is the idea that because Rollup concatenates it into a single file, it would still be smaller in the host application, because it only has 1 webpack-module-wrapper chunk of code?

That's the reason - compiling into a "flat bundle" (single file output) will result in overall smaller size in the host application.

Reply:
Ok! Cool. Truthfully my feelings are still a bit mixed, but ultimately I think the size benefit outweighs the small DX cost. This does appear to be conventional with other libs, noticed styled-components does this.

Thanks for being patient with my questions about this!

Reply:
Not a problem, Im happy to answer any questions regarding those topics. For better DX we could emit sourcemaps too, but actually Im not a big fan of them as they obscure identifiers names and checking values of various expressions quickly in the console becomes much harder. Your call on all of this anyway :)